src/SDKs/Reservations/Management.Reservations/Generated/Models/Patch.cs

```diff
@@ -65,5 +67,11 @@ public Patch(string appliedScopeType = default(string), IList<string> appliedSco
         [JsonProperty(PropertyName = "properties.instanceFlexibility")]
         public string InstanceFlexibility { get; set; }
 
+        /// <summary>
+        /// Gets or sets name of the Reservation
+        /// </summary>
+        [JsonProperty(PropertyName = "properties.name")]
+        public string Name { get; set; }
+
```

Comment:
These look like additive changes.
Please update package version, assembly version and package realease notes in the csproj and AssemblyInfo.cs file

Reply:
Sure.  Currently, it is 1.1.0-preview. Should it be updated to 1.2.0-preview?
AssemblyInfo.cs should be 1.0.2.0 ?

Reply:
I also saw the AssemblyFileVersion currenlty as 1.8.0.0. Should I update that too.

Reply:
`1.8.0-preview` is the last published version.
Please update package version to `1.9.0-preview` and `AssemblyFileVersion`  in `AssemblyInfo.cs` to `1.9.0.0`

Reply:
Done.